package.json

```diff
@@ -43,5 +43,8 @@
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-react": "^6.16.0",
     "babel-preset-stage-0": "^6.16.0"
+  },
+  "dependencies": {
+    "prop-types": "^15.5.8"
```

Comment:
What about user using React version < 15.5? Will they have both standalone `prop-types` and the build in `React.PropTypes`?